### DISCUSSION TITLE
uc mode is detected by Twitch.tv login

### DISCUSSION THREAD

**Discussion by Magic4k** [seleniumbase/SeleniumBase#2734] [Q&A] [answered]
when using a uc mode and trying to make a basic sign in to (twitch.tv) ,writing username and password 
normally then trying to sign in but it gives an error message "Your browser is not currently supported. Please use a recommended browser or learn more here." so is it a detection ? or not and what is the solution ?
I think I tried everything but nothing worked so please help me

**Comment by mdmintz** [comment, marked as answer]
In order to remain undetected in UC Mode when clicking a button, you have to use the `uc_click()` method. Eg.

```python
sb.driver.uc_click('button[data-a-target="passport-login-button"]')
```

**Reply by Magic4k** [reply]
it worked thanks alot ♥
